source-kinesis/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# Build Stage
+################################################################################
+FROM golang:1.17-buster as builder
+
+WORKDIR /builder
+
+# Compile-time dependencies
```

Comment:
This is intended for removal ?

Reply:
Yeah, it wasn't intended to live on in any final Dockerfiles that didn't need any dependencies. I just had it as a placeholder.